draft-ietf-lake-edhoc.md

```diff
@@ -1337,7 +1338,7 @@ When parsing a received EDHOC message, implementations MUST terminate the protoc
 
 ## EDHOC Exporter Label Registry {#exporter-label}
 
-IANA has created a new registry titled "EDHOC Exporter Label" under the new group name "Ephemeral Diffie-Hellman Over COSE (EDHOC)". The registration procedure is "Expert Review". The columns of the registry are Label and Description. Label is a uint. Description is a text string. The initial contents of the registry are:
+IANA has created a new registry titled "EDHOC Exporter Label" under the new registry group "Ephemeral Diffie-Hellman Over COSE (EDHOC)". The registration procedure is "Expert Review" {{RFC8126}}. The columns of the registry are Label and Description. Label is a uint. Description is a text string. The initial contents of the registry are:
```

Comment:
I don't understand why group name should be changed to registry group

  Unfortunately, we have been inconsistent in how we refer to these
   entities.  The group names, as they are referred to here, have been
   variously called "protocol category groups", "groups", "top-level
   registries", or just "registries".  The registries under them have
   been called "registries" or "sub-registries".

But I don't care.